azure-devops/azext_devops/dev/repos/pull_request.py

```diff
@@ -299,6 +305,10 @@ def update_pull_request(id, title=None, description=None, auto_complete=None,  #
             pr.auto_complete_set_by = IdentityRef(id=resolve_identity_as_id(ME, organization))
         else:
             pr.auto_complete_set_by = IdentityRef(id=EMPTY_UUID)
+    if publish and existing_pr.is_draft:
```

Comment:
what if both `publish` and `mark_as_draft` is true at the same time?

Reply:
Will handle this. I am waiting for closing the other comment where you suggested to get rid of two parameters in favour of one. Keeping this open till we close that one.